NEWS.md

```diff
@@ -75,17 +75,22 @@
   more information.
 
 * Double-dispatch methods for `vec_ptype2()` and `vec_cast()` are now
-  easier to implement. vctrs takes care of implementing the default
-  and unspecified methods. If you have implemented these methods, they
-  can now be removed.
+  easier to implement. They no longer need any the boiler plate.
+  Implementing a method for classes `foo` and `bar` is now as simple as:
 
-  Also methods no longer have to worry about order of dispatch, for
-  instance it is no longer needed to supply a second argument as in
-  `UseMethod("vec_ptype2.mytype", y)`.
+  ```
+  #' @export
+  vec_ptype2.foo.bar <- function(x, y, ...) new_foo()
+  ```
+
+  vctrs also takes care of implementing the default and unspecified
+  methods. If you have implemented these methods, they are no longer
+  called and can now be removed.
```

Comment:
But if you do have them, everything will continue to work, right?

Reply:
Right.